docs/development.rst

```diff
@@ -14,17 +14,48 @@ I am the primary author of Hypothesis.
 5. Planning the general roadmap of the project
 6. Doing sponsored development on tasks that are too large or in depth for other people to take on
 
-So all new features must either be sponsored or implemented by someone else. That being said, I take a fairly active
-role in shepherding pull requests and helping people write a new feature (see `the
-contributing guidelines <https://github.com/HypothesisWorks/hypothesis-python/blob/master/CONTRIBUTING.rst>`_ for
-details and `this pull request
-<https://github.com/HypothesisWorks/hypothesis-python/pull/154>`_ for an example of how the process goes). This isn't
+So all new features must either be sponsored or implemented by someone else.
+That being said, I take a fairly active role in shepherding pull requests and
+helping people write a new feature (see :gh-file:`CONTRIBUTING.rst` for
+details and :pull:`154` for an example of how the process goes). This isn't
 "patches welcome", it's "I will help you write a patch".
 
-All enhancement tickets on GitHub are tagged with either `help-wanted <https://github.com/HypothesisWorks/hypothesis-python/labels/help-wanted>`_
-if I think they're viable for someone else to pick up or `for-a-modest-fee <https://github.com/HypothesisWorks/hypothesis-python/labels/for-a-modest-fee>`_ if
+All enhancement tickets on GitHub are tagged with either :gh-link:`help-wanted <labels/help-wanted>`
+if I think they're viable for someone else to pick up or :gh-link:`for-a-modest-fee <labels/for-a-modest-fee>` if
```

Comment:
I actually think we should just drop this section. It's not really experienced much uptake on either of these, and I've not been good at tagging issues with them for a while.